setup.cfg

```diff
@@ -26,10 +26,8 @@ package_dir=
 packages = find:
 # TODO: This should be treated as a legacy workaround and eventually removed
 scripts =
-    src/DIRAC/Core/scripts/genAllCAs.sh
     src/DIRAC/Core/scripts/dirac-cert-convert.sh
```

Comment:
This stays?

Reply:
Yes, because dirac-cert-convert.py still calls the shell openssl command (M2Crypto doesn't seem to work with p12). Not sure what is better .py or .sh here, so I left .sh.

Reply:
You're right I have to delete it, fixed.